test/spacesuit_cors_middleware_test.exs

```diff
@@ -0,0 +1,156 @@
+defmodule SpacesuitCorsMiddlewareTest do
```

Comment:
Looks to me like there are various missing tests around some settings that appear in the code and not in the tests.